dbt/adapters/bigquery/impl.py

```diff
@@ -907,3 +909,14 @@ def python_submission_helpers(self) -> Dict[str, Type[PythonJobHelper]]:
             "cluster": ClusterDataprocHelper,
             "serverless": ServerlessDataProcHelper,
         }
+
+    @classmethod
+    def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
+        if constraint.type != ConstraintType.not_null:
+            return ""  # Only not null column constraints are supported by BigQuery
+        else:
```

Comment:
It also supports primary and foreign keys now.  See https://github.com/dbt-labs/dbt-bigquery/issues/568

Reply:
Supported PKs and FKs in last two commits.